src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1489,5 +1492,40 @@ public String toVariableNameString(EnchantmentOffer eo) {
 				.since("INSERT VERSION")
 				.requiredPlugins("Paper 1.16+"));
 		}
+
+		Classes.registerClass(new ClassInfo<>(BossBar.class, "bossbar")
```

Comment:
Bossbars can be serialized https://hub.spigotmc.org/javadocs/spigot/org/bukkit/boss/KeyedBossBar.html

In this case of this pull, it could go under the player serialization.

Reply:
they can be, but I honestly don't think it's actually a desirable feature. imo they are usually dynamic in nature so serializing them doesn't have much utility.

Reply:
As i've had them in Skellett for a long time, I can confirm people do want this feature, and Spigot has KeyableBossBars